doc/src/properties/atom.toml

```diff
@@ -0,0 +1,22 @@
+[is_hetatm]
+type = "bool"
+
+PDB = """When reading, **is_hetatm** is set to ``true`` for atoms defined with a
+``HETATM`` record, and ``false`` for atoms defined with an ``ATOM`` record.
+When writing, **is_hetatm** is used to determine whether to emit an ``HETATM``
+or an ``ATOM`` record. If the property is not set, ``HETATM`` is used."""
+
+MMTF = "**TODO**"
+
+mmCIF = """When reading, **is_hetatm** is set to ``true`` when
+``_atom_site.group_PDB`` is ``HETATM``, and ``false`` when it is ``ATOM``. When
+writing, **is_hetatm** is used to determine whether to use ``HETATM`` or
+``ATOM`` for ``_atom_site.group_PDB``. If the property is not set, ``ATOM`` is
```

Comment:
So PDB uses `HETATM` as a default, and mmCIF uses `ATOM` as a default. Do we want to have both of them use the same default value ?

`HETATM` was picked for PDB because we have no way (except the `is_hetatm` propety) to know whether the atom is an hetero atom from the PDB point of view.

Reply:
Technically speaking, mmCIF differentiates between `ATOM` and `HETATM` via the connectivity of the residue to other residues (the `MMTF` code that assigns `HETATM` checks connectivity types). I think that the same goes for PDB.

That being said, we can switch the mmCIF default to `HETATM` or attempt to see if the residue is in the PDB connectivity table and only include residues in this table that are the correct connectivity type (IE amino acids and nucleotides).

Reply:
I opened #210 to discuss this.